src/mixer/samplerbank.cpp

```diff
@@ -38,23 +50,30 @@ void SamplerBank::slotSaveSamplerBank(double v) {
         return;
     }
 
-    const QString xmlSuffix = QStringLiteral(".xml");
-    QString fileFilter = tr("Mixxx Sampler Banks (*%1)").arg(xmlSuffix);
-    QString samplerBankPath = QFileDialog::getSaveFileName(nullptr,
+    QString lastImportExportDirectory = m_pConfig->getValue(
+            kConfigkeyLastImportExportDirectory,
+            // When Mixxx exits samplers are auto-exported to the config directory.
+            // Let's choose a different location to avoid confusion.
+            QStandardPaths::writableLocation(QStandardPaths::MusicLocation));
+
+    // Open a dialog to let the user choose the file location for crate export.
+    // The location is set to the last used directory for import/export and the file
+    // name to the playlist name.
+    const QString samplerBankPath = getFilePathWithVerifiedExtensionFromFileDialog(
             tr("Save Sampler Bank"),
-            QString(),
-            fileFilter,
-            &fileFilter);
-    if (samplerBankPath.isNull() || samplerBankPath.isEmpty()) {
+            lastImportExportDirectory,
+            kSamplerFileType,
```

Comment:
```suggestion
            tr(kSamplerFileType),
```

Reply:
...and every similiar occurence I assume.